ch390/src/esp_eth_phy_ch390.c

```diff
@@ -126,10 +139,10 @@ static esp_err_t ch390_loopback(esp_eth_phy_t *phy, bool enable)
 
     if (enable) {
         bmcr.en_loopback = 1;
-        phy_ctl1.pma_lpbk = 1;
+        phy_ctl1.remote_lpbk = 1;
```

Comment:
Note to myself. Need to figure out, what kind of loopback it is. If it is far-end (as usually called by other vendors), it't not indented to be enabled here.

Reply:
Reply from WCH:
"_IEEE802.3 standard interface MII register 0x0 (Control Register) bit 14 sets the loopback. PCS, PMA and remote belong to 4 different loopbacks. They belong to different layers. (more subdivided layers)

As mentioned above, different loopbacks are independent. You only need to enable one. It is recommended to use the loopback method disclosed in DS1, which is the loopback set by IEEE802.3 standard interface MII register 0x0 (Control Register) bit 14.

Other loopback methods are mainly used for hardware testing.

“Remote loopback” is the same as ”Far-end loopback“ mentioned by other vendors._"

So you don't need to enable it. It seems IEEE802.3 loopback only should be OK. You can use the common `phy_802_3` function.

Reply:
@SergeyKharenko just a gentle reminder. Could you please resolve this comment based on feedback from WCH?

Reply:
> @SergeyKharenko just a gentle reminder. Could you please resolve this comment based on feedback from WCH?

Sorry for my late reply~ I have been busy with my academic report recently😭
The commit has been submitted~ I fully delete the definition of phy_ctrl1_reg_t and its usage.